# Profitable App Profiles for the App Store and Google Play Markets 
This project is an analysis of apps from Google Play and App Store that are free to download and install.

Our goal is to understand what type of apps attract more users while the number of users determines the revenue of any of those apps as the main source of revenue consists of in-app ads.

## Opening and Exploring the data

### Opening the two datasets
The first one is a dataset with approximately seven thousand apps from Apple Store. It can be found in [this link](https://www.kaggle.com/ramamet4/app-store-apple-data-set-10k-apps).

The second one is a dataset with approximately ten thousand apps from Google Play. It can be found in [this link](https://www.kaggle.com/lava18/google-play-store-apps).


In [1]:
import csv
with open('AppleStore.csv', 'r') as apple_file:
    apple_read = csv.reader(apple_file)
    ios = list(apple_read)
    ios_header = ios[0]
    ios = ios[1:]
    
with open('googleplaystore.csv', 'r') as google_file:
    google_read = csv.reader(google_file)
    android = list(google_read)
    android_header = android[0]
    android = android[1:]

### Exploring the data

In [2]:
def explore_data(dataset, start, end, rows_and_columns = False):
    dataset_slice = dataset[start:end]
    for row in dataset_slice:
        print(row)
        print('\n')
    if rows_and_columns:
        print("Number of rows:", len(dataset))
        print("Number of columns:", len(dataset[0]))

In [3]:
print(20*'-', 'iOS table', 20*'-')
print(ios_header, '\n')
print(explore_data(ios, 0, 3, True))
print(20*'-', 'Android table', 20*'-')
print(android_header, '\n')
print(explore_data(android, 0, 3, True))

-------------------- iOS table --------------------
['id', 'track_name', 'size_bytes', 'currency', 'price', 'rating_count_tot', 'rating_count_ver', 'user_rating', 'user_rating_ver', 'ver', 'cont_rating', 'prime_genre', 'sup_devices.num', 'ipadSc_urls.num', 'lang.num', 'vpp_lic'] 

['284882215', 'Facebook', '389879808', 'USD', '0.0', '2974676', '212', '3.5', '3.5', '95.0', '4+', 'Social Networking', '37', '1', '29', '1']


['389801252', 'Instagram', '113954816', 'USD', '0.0', '2161558', '1289', '4.5', '4.0', '10.23', '12+', 'Photo & Video', '37', '0', '29', '1']


['529479190', 'Clash of Clans', '116476928', 'USD', '0.0', '2130805', '579', '4.5', '4.5', '9.24.12', '9+', 'Games', '38', '5', '18', '1']


Number of rows: 7197
Number of columns: 16
None
-------------------- Android table --------------------
['App', 'Category', 'Rating', 'Reviews', 'Size', 'Installs', 'Type', 'Price', 'Content Rating', 'Genres', 'Last Updated', 'Current Ver', 'Android Ver'] 

['Photo Editor & Candy Camera &

For our analysis, columns that could help are 'track_name', 'currency', 'price', 'rating_count_tot', 'rating_count_ver', 'cont_rating' and 'prime_genre' in iOs table. Description of the column names can be found in the link of the database.

In Android table, columns that could help are 'App', 'Category', 'Reviews', 'Installs', 'Type', 'Price' and 'Genres'.

## Deleting wrong data

The Google Play Store Apps data set has a discussion section and one of the discussions indicates that one column data is missing in the row 10472.

In [4]:
print(android[0], '\nNumber of columns:', len(android[0]))
print(android[10472], '\nNumber of columns:', len(android[10472]))

['Photo Editor & Candy Camera & Grid & ScrapBook', 'ART_AND_DESIGN', '4.1', '159', '19M', '10,000+', 'Free', '0', 'Everyone', 'Art & Design', 'January 7, 2018', '1.0.0', '4.0.3 and up'] 
Number of columns: 13
['Life Made WI-Fi Touchscreen Photo Frame', '1.9', '19', '3.0M', '1,000+', 'Free', '0', 'Everyone', '', 'February 11, 2018', '1.0.19', '4.0 and up'] 
Number of columns: 12


We can see that the number of columns are different and, in order to solve this problem, we will delete this row.

In [5]:
print(len(android))
del android[10472]
print(len(android))

10841
10840


We can see if there is another row like the one we deleted in both datasets.

In [6]:
for row in range(len(android)):
    n_col_header = len(android_header)
    n_col_table = len(android[row])
    if n_col_header != n_col_table:
        print('row: ', row, '\n')
        print(android[row])

for row in range(len(ios)):
    n_col_header = len(ios_header)
    n_col_table = len(ios[row])
    if n_col_header != n_col_table:
        print('row: ', row, '\n')
        print(ios[row])

## Removing Duplicate Entries

Another discussion of the data set is about duplicate entries. For example, the application instragram in the android data set:

In [7]:
for row in android:
    if row[0] == 'Instagram':
        print(row)

['Instagram', 'SOCIAL', '4.5', '66577313', 'Varies with device', '1,000,000,000+', 'Free', '0', 'Teen', 'Social', 'July 31, 2018', 'Varies with device', 'Varies with device']
['Instagram', 'SOCIAL', '4.5', '66577446', 'Varies with device', '1,000,000,000+', 'Free', '0', 'Teen', 'Social', 'July 31, 2018', 'Varies with device', 'Varies with device']
['Instagram', 'SOCIAL', '4.5', '66577313', 'Varies with device', '1,000,000,000+', 'Free', '0', 'Teen', 'Social', 'July 31, 2018', 'Varies with device', 'Varies with device']
['Instagram', 'SOCIAL', '4.5', '66509917', 'Varies with device', '1,000,000,000+', 'Free', '0', 'Teen', 'Social', 'July 31, 2018', 'Varies with device', 'Varies with device']


And the app Mannequin Challenge in the ios data set:

In [8]:
for row in ios:
    if row[1] == 'Mannequin Challenge':
        print(row)

['1173990889', 'Mannequin Challenge', '109705216', 'USD', '0.0', '668', '87', '3.0', '3.0', '1.4', '9+', 'Games', '37', '4', '1', '1']
['1178454060', 'Mannequin Challenge', '59572224', 'USD', '0.0', '105', '58', '4.0', '4.5', '1.0.1', '4+', 'Games', '38', '5', '1', '1']


In [9]:
app_duplicate_android = []
app_unique_android = []
for row in android:
    if row[0] in app_unique_android:
        app_duplicate_android.append(row[0])
    else:
        app_unique_android.append(row[0])

print("Number of android duplicate apps:", len(app_duplicate_android))

Number of android duplicate apps: 1181


In [10]:
app_duplicate_ios = []
app_unique_ios = []
for row in ios:
    if row[1] in app_unique_ios:
        app_duplicate_ios.append(row[1])
    else:
        app_unique_ios.append(row[1])

print("Number of ios duplicate apps:", len(app_duplicate_ios))

Number of ios duplicate apps: 2


We want to remove these duplicate entries, but not randomly because we can see in the Instagram example that the values in the fourth column (number of users reviews for the app) are different. 
These different numbers show the data was collected at different times. Therefore we will keep the row with the highest number of reviews since it should be the more recent data.

To do that:
* We will create a dictionary with the name of each app as a key and its highest number of reviews as value.
* We will create a new data set with only the rows with the highest number of reviews.

In [11]:
type(android[0][3])

str

In [12]:
reviews_max_android = {}

for row in android:
    name = row[0]
    n_reviews = float(row[3])
    if name in reviews_max_android and reviews_max_android[name] < n_reviews:
        reviews_max_android[name] = n_reviews
    elif name not in reviews_max_android:
        reviews_max_android[name] = n_reviews

print("Expected number of entries: ", len(android) - len(app_duplicate_android), '\n')
print("Number of entries in the dictionary: ", len(reviews_max_android))

Expected number of entries:  9659 

Number of entries in the dictionary:  9659


In [13]:
rating_max_ios = {}

for row in ios:
    name = row[1]
    n_rating = float(row[5])
    if name in rating_max_ios and rating_max_ios[name] < n_rating:
        rating_max_ios[name] = n_rating
    elif name not in rating_max_ios:
        rating_max_ios[name] = n_rating

print("Expected number of entries: ", len(ios) - len(app_duplicate_ios), '\n')
print("Number of entries in the dictionary: ", len(rating_max_ios))

Expected number of entries:  7195 

Number of entries in the dictionary:  7195


In [14]:
android_clean = []
already_added = []

for row in android:
    name = row[0]
    n_reviews = float(row[3])
    if n_reviews == reviews_max_android[name] and name not in already_added:
        android_clean.append(row)
        already_added.append(name)

print("Number of entries in the new data set: ", len(android_clean))


Number of entries in the new data set:  9659


## Removing non-english apps

If we explore the data, we will find that the datasets have apps with names for not only an English-speaking audience.

In [15]:
print(ios[813][1])
print(android_clean[4412][0])

爱奇艺PPS -《欢乐颂2》电视剧热播
中国語 AQリスニング


And as we would like to analyze only apps for an English-speaking audience, we will remove them.

We built this function below to find out if there is only letters from the English alphabet in text. 
The only problem is that emojis and other symbols aren't considered an English text in this function.

In [16]:
def is_english(string):
    for letter in string:
        if ord(letter) > 127:
            return False
    return True

print(is_english('Instagram'))
print(is_english('爱奇艺PPS -《欢乐颂2》电视剧热播'))
print(is_english('Docs To Go™ Free Office Suite'))
print(is_english('Instachat 😜'))

True
False
False
False


To minimize the impact of data loss, we'll only remove an app if its name has more than three non-ASCII characters:

In [17]:
def is_english(string):
    non_ascii = 0 
    for letter in string:
        if ord(letter) > 127:
            non_ascii += 1
    return False if non_ascii > 3 else True
print(is_english('Instagram'))
print(is_english('爱奇艺PPS -《欢乐颂2》电视剧热播'))
print(is_english('Docs To Go™ Free Office Suite'))
print(is_english('Instachat 😜'))

True
False
True
True


Below, we use the function to filter the non-English apps from the data sets:

In [18]:
english_android = []
english_ios = []

for row in android_clean:
    name = row[0]
    if is_english(name):
        english_android.append(row)

for row in ios:
    name = row[1]
    if is_english(name):
        english_ios.append(row)
    
explore_data(english_android, 0, 3, True)
print(50 * ('-'))
explore_data(english_ios, 0, 3, True)

['Photo Editor & Candy Camera & Grid & ScrapBook', 'ART_AND_DESIGN', '4.1', '159', '19M', '10,000+', 'Free', '0', 'Everyone', 'Art & Design', 'January 7, 2018', '1.0.0', '4.0.3 and up']


['U Launcher Lite – FREE Live Cool Themes, Hide Apps', 'ART_AND_DESIGN', '4.7', '87510', '8.7M', '5,000,000+', 'Free', '0', 'Everyone', 'Art & Design', 'August 1, 2018', '1.2.4', '4.0.3 and up']


['Sketch - Draw & Paint', 'ART_AND_DESIGN', '4.5', '215644', '25M', '50,000,000+', 'Free', '0', 'Teen', 'Art & Design', 'June 8, 2018', 'Varies with device', '4.2 and up']


Number of rows: 9614
Number of columns: 13
--------------------------------------------------
['284882215', 'Facebook', '389879808', 'USD', '0.0', '2974676', '212', '3.5', '3.5', '95.0', '4+', 'Social Networking', '37', '1', '29', '1']


['389801252', 'Instagram', '113954816', 'USD', '0.0', '2161558', '1289', '4.5', '4.0', '10.23', '12+', 'Photo & Video', '37', '0', '29', '1']


['529479190', 'Clash of Clans', '116476928', 'USD', '0.0', 

## Isolating the free apps

As we mentioned in the introduction, we only build apps that are free to download and install. Our data sets contain both free and non-free apps, and we'll need to isolate only the free apps for our analysis. 

In [19]:
free_android = []
free_ios = []

for row in english_android:
    if row[7] == '0':
        free_android.append(row)

for row in english_ios:
    if row[4] == '0.0':
        free_ios.append(row)
        
print('Remaining free android apps: ', len(free_android))
print('Remaining free ios apps: ',len(free_ios))

Remaining free android apps:  8864
Remaining free ios apps:  3222


## Most common apps by genre

As we mentioned in the introduction, our aim is to determine the kinds of apps that are likely to attract more users because our revenue is highly influenced by the number of people using our apps. We need to find app profiles that are successful on both markets, Google Play and App Store.

Let's begin the analysis by getting a sense of what are the most common genres for each market. For this, we'll need to build frequency tables for a few columns in our datasets.

We'll build two functions we can use to analyze the frequency tables:

* One function to generate frequency tables that show percentages
* Another function we can use to display the percentages in a descending order

In [20]:
# index is the index of the column
def freq_table(dataset, index):
    freq = {}
    total = 0
    for row in dataset:
        genre = row[index]
        total += 1
        if genre not in freq:
            freq[genre] = 1
        else:
            freq[genre] += 1
    table_percentage = {}
    for key in freq:
        table_percentage[key] = freq[key] / total * 100
        
    return table_percentage

In [21]:
def display_table(dataset, index):
    table = freq_table(dataset, index)
    table_display = []
    for val_key_tuple in zip(table.values(), table.keys()):
        table_display.append(val_key_tuple)
    
    table_sorted = sorted(table_display, reverse = True)
    for tup in table_sorted:
        print(f"{tup[1]:19}: {tup[0]}")

In [22]:
display_table(free_android, 1) #category

FAMILY             : 18.907942238267147
GAME               : 9.724729241877256
TOOLS              : 8.461191335740072
BUSINESS           : 4.591606498194946
LIFESTYLE          : 3.9034296028880866
PRODUCTIVITY       : 3.892148014440433
FINANCE            : 3.7003610108303246
MEDICAL            : 3.531137184115524
SPORTS             : 3.395758122743682
PERSONALIZATION    : 3.3167870036101084
COMMUNICATION      : 3.2378158844765346
HEALTH_AND_FITNESS : 3.0798736462093865
PHOTOGRAPHY        : 2.944494584837545
NEWS_AND_MAGAZINES : 2.7978339350180503
SOCIAL             : 2.6624548736462095
TRAVEL_AND_LOCAL   : 2.33528880866426
SHOPPING           : 2.2450361010830324
BOOKS_AND_REFERENCE: 2.1435018050541514
DATING             : 1.861462093862816
VIDEO_PLAYERS      : 1.7937725631768955
MAPS_AND_NAVIGATION: 1.3989169675090252
FOOD_AND_DRINK     : 1.2409747292418771
EDUCATION          : 1.1620036101083033
ENTERTAINMENT      : 0.9589350180505415
LIBRARIES_AND_DEMO : 0.9363718411552346
AUTO_AND_V

There are not that many apps designed for fun, and it seems that a good number of free English apps are designed for practical purposes (family, tools, business, lifestyle, productivity, etc.). However, we can see that the family category means mostly games for kids.

In [23]:
display_table(free_android, 9) #genre

Tools              : 8.449909747292418
Entertainment      : 6.069494584837545
Education          : 5.347472924187725
Business           : 4.591606498194946
Productivity       : 3.892148014440433
Lifestyle          : 3.892148014440433
Finance            : 3.7003610108303246
Medical            : 3.531137184115524
Sports             : 3.463447653429603
Personalization    : 3.3167870036101084
Communication      : 3.2378158844765346
Action             : 3.1024368231046933
Health & Fitness   : 3.0798736462093865
Photography        : 2.944494584837545
News & Magazines   : 2.7978339350180503
Social             : 2.6624548736462095
Travel & Local     : 2.3240072202166067
Shopping           : 2.2450361010830324
Books & Reference  : 2.1435018050541514
Simulation         : 2.0419675090252705
Dating             : 1.861462093862816
Arcade             : 1.8501805054151623
Video Players & Editors: 1.7712093862815883
Casual             : 1.7599277978339352
Maps & Navigation  : 1.3989169675090252
Food &

We can notice that the Genres column has more categories that the category column.

In [24]:
display_table(free_ios, 11)

Games              : 58.16263190564867
Entertainment      : 7.883302296710118
Photo & Video      : 4.9658597144630665
Education          : 3.662321539416512
Social Networking  : 3.2898820608317814
Shopping           : 2.60707635009311
Utilities          : 2.5139664804469275
Sports             : 2.1415270018621975
Music              : 2.0484171322160147
Health & Fitness   : 2.0173805090006205
Productivity       : 1.7380509000620732
Lifestyle          : 1.5828677839851024
News               : 1.3345747982619491
Travel             : 1.2414649286157666
Finance            : 1.1173184357541899
Weather            : 0.8690254500310366
Food & Drink       : 0.8069522036002483
Reference          : 0.5586592178770949
Business           : 0.5276225946617008
Book               : 0.4345127250155183
Navigation         : 0.186219739292365
Medical            : 0.186219739292365
Catalogs           : 0.12414649286157665


We can see that among the free English apps, more than a half (58.16%) are games. Entertainment apps are close to 8%, followed by photo and video apps, which are close to 5%. Only 3.66% of the apps are designed for education, followed by social networking apps which amount for 3.29% of the apps in our data set.

The impression is that App Store is dominated by apps that are designed for fun in the part containing free English apps. However, the fact that fun apps are the most numerous doesn't also imply that they also have the greatest number of users.

Up to date, we found that the App Store is dominated by apps designed for fun while Google Play is by practical and for-fun apps. 

## Most popular apps by genre

### On App Store 

We can calculate the average number of installs for each app genre to find the most popular genres. For the App Store dataset, we will take the total number of user ratings in the rating_count_tot column as a proxy for the number of installs for each app genre.

In [25]:
unique_genres_ios = freq_table(free_ios, 11)

for genre in unique_genres_ios:
    total = 0
    len_genre = 0
    for app in free_ios:
        app_genre = app[11]
        if app_genre == genre:
            ratings = app[5]
            total += float(ratings)
            len_genre += 1
    
    avg_genre = total / len_genre
    print(f"{genre :17}: {avg_genre:,.2f}")

Social Networking: 71,548.35
Photo & Video    : 28,441.54
Games            : 22,788.67
Music            : 57,326.53
Reference        : 74,942.11
Health & Fitness : 23,298.02
Weather          : 52,279.89
Utilities        : 18,684.46
Travel           : 28,243.80
Shopping         : 26,919.69
News             : 21,248.02
Navigation       : 86,090.33
Lifestyle        : 16,485.76
Entertainment    : 14,029.83
Food & Drink     : 33,333.92
Sports           : 23,008.90
Book             : 39,758.50
Finance          : 31,467.94
Education        : 7,003.98
Productivity     : 21,028.41
Business         : 7,491.12
Catalogs         : 4,004.00
Medical          : 612.00


Navigation apps are the most popular on average influenced by Waze and Google Maps.

In [26]:
for app in free_ios:
    if app[11] == 'Navigation':
        print(f"{app[1]:50}: {int(app[5]):,}")

Waze - GPS Navigation, Maps & Real-time Traffic   : 345,046
Google Maps - Navigation & Transit                : 154,911
Geocaching®                                       : 12,811
CoPilot GPS – Car Navigation & Offline Maps       : 3,582
ImmobilienScout24: Real Estate Search in Germany  : 187
Railway Route Search                              : 5


Other popular apps by genre are Reference, Social Networking, and Music. These apps are also influenced by giants like Facebook and Pandora.

In [27]:
for app in free_ios:
    if app[11] in ['Reference', 'Social Networking', 'Music']:
        print(f"{app[1]:50}: {int(app[5]):,}")

Facebook                                          : 2,974,676
Pandora - Music & Radio                           : 1,126,879
Pinterest                                         : 1,061,624
Bible                                             : 985,920
Spotify Music                                     : 878,563
Shazam - Discover music, artists, videos & lyrics : 402,925
Skype for iPhone                                  : 373,519
Messenger                                         : 351,466
Tumblr                                            : 334,293
iHeartRadio – Free Music & Radio Stations         : 293,228
WhatsApp Messenger                                : 287,589
Kik                                               : 260,965
Dictionary.com Dictionary & Thesaurus             : 200,047
ooVoo – Free Video Call, Text and Voice           : 177,501
TextNow - Unlimited Text + Calls                  : 164,963
Viber Messenger – Text & Call                     : 164,249
SoundCloud - Music & Audio        

These apps influenced by giants seem to affect our average and our sight of genre popularity. We can redo the average calculation without these extremely popular apps with more than 300,000 ratings.

In [28]:
for genre in unique_genres_ios:
    new_total = 0
    new_len_genre = 0
    for app in free_ios:
        app_genre = app[11]
        if app_genre == genre:
            ratings = float(app[5])
            if ratings < 300000:
                new_total += ratings
                new_len_genre += 1
    
    new_avg_genre = new_total / new_len_genre
    print(f"{genre :17}: {new_avg_genre:,.2f}")

Social Networking: 24,639.08
Photo & Video    : 13,070.78
Games            : 13,759.67
Music            : 21,828.32
Reference        : 21,355.18
Health & Fitness : 10,044.92
Weather          : 35,859.67
Utilities        : 12,925.01
Travel           : 17,527.36
Shopping         : 22,210.54
News             : 13,323.98
Navigation       : 34,299.20
Lifestyle        : 9,956.10
Entertainment    : 12,864.56
Food & Drink     : 22,513.04
Sports           : 23,008.90
Book             : 39,758.50
Finance          : 31,467.94
Education        : 7,003.98
Productivity     : 21,028.41
Business         : 7,491.12
Catalogs         : 4,004.00
Medical          : 612.00


Now other genres seem popular, like weather, book, and finance. Especially the book genre, since it is the most popular on this new average.

### On Google Play

In the Google Play dataset, we have a column with data about the number of installs. The data in this column are categories, and each one provides a minimum value.

In [29]:
display_table(free_android, 5)

1,000,000+         : 15.726534296028879
100,000+           : 11.552346570397113
10,000,000+        : 10.548285198555957
10,000+            : 10.198555956678701
1,000+             : 8.393501805054152
100+               : 6.915613718411552
5,000,000+         : 6.825361010830325
500,000+           : 5.561823104693141
50,000+            : 4.7721119133574
5,000+             : 4.512635379061372
10+                : 3.5424187725631766
500+               : 3.2490974729241873
50,000,000+        : 2.3014440433213
100,000,000+       : 2.1322202166064983
50+                : 1.917870036101083
5+                 : 0.78971119133574
1+                 : 0.5076714801444043
500,000,000+       : 0.2707581227436823
1,000,000,000+     : 0.22563176895306858
0+                 : 0.04512635379061372
0                  : 0.01128158844765343


Since we only want to find apps that attract more users, this is not a problem. 
Before calculating the average as in the App Store dataset, we need to convert those values to a valid number type.

In [30]:
unique_categories = freq_table(free_android, 1)
for category in unique_categories:
    total = 0
    len_category = 0
    for app in free_android:
        category_app = app[1]
        if category_app == category:
            n_installs = app[5]
            n_installs = n_installs.replace('+','')
            n_installs = n_installs.replace(',','')
            total += float(n_installs)
            len_category += 1
    avg_category = total / len_category
    print(f'{category :20}: {avg_category :,.2f}')

ART_AND_DESIGN      : 1,986,335.09
AUTO_AND_VEHICLES   : 647,317.82
BEAUTY              : 513,151.89
BOOKS_AND_REFERENCE : 8,767,811.89
BUSINESS            : 1,712,290.15
COMICS              : 817,657.27
COMMUNICATION       : 38,456,119.17
DATING              : 854,028.83
EDUCATION           : 1,833,495.15
ENTERTAINMENT       : 11,640,705.88
EVENTS              : 253,542.22
FINANCE             : 1,387,692.48
FOOD_AND_DRINK      : 1,924,897.74
HEALTH_AND_FITNESS  : 4,188,821.99
HOUSE_AND_HOME      : 1,331,540.56
LIBRARIES_AND_DEMO  : 638,503.73
LIFESTYLE           : 1,437,816.27
GAME                : 15,588,015.60
FAMILY              : 3,695,641.82
MEDICAL             : 120,550.62
SOCIAL              : 23,253,652.13
SHOPPING            : 7,036,877.31
PHOTOGRAPHY         : 17,840,110.40
SPORTS              : 3,638,640.14
TRAVEL_AND_LOCAL    : 13,984,077.71
TOOLS               : 10,801,391.30
PERSONALIZATION     : 5,201,482.61
PRODUCTIVITY        : 16,787,331.34
PARENTING           : 542,

On average, communication apps have the most installs, and this number is heavily skewed up by a few apps, just like in the App Store dataset. We see the same pattern for the video players category, social apps, photography apps, productivity apps and other.

These app genres seem more popular than they really are and dominated by a few giants who are hard to compete.

In [31]:
for app in free_android:
    if app[1] == 'COMMUNICATION':
         print(f"{app[0]:50}: {app[5]}")

WhatsApp Messenger                                : 1,000,000,000+
Messenger for SMS                                 : 10,000,000+
My Tele2                                          : 5,000,000+
imo beta free calls and text                      : 100,000,000+
Contacts                                          : 50,000,000+
Call Free – Free Call                             : 5,000,000+
Web Browser & Explorer                            : 5,000,000+
Browser 4G                                        : 10,000,000+
MegaFon Dashboard                                 : 10,000,000+
ZenUI Dialer & Contacts                           : 10,000,000+
Cricket Visual Voicemail                          : 10,000,000+
TracFone My Account                               : 1,000,000+
Xperia Link™                                      : 10,000,000+
TouchPal Keyboard - Fun Emoji & Android Keyboard  : 10,000,000+
Skype Lite - Free Video Call & Chat               : 5,000,000+
My magenta                               

The books and reference genre is not the most popular, but it still looks popular. The same happens with the weather genre, and it's interesting because we also found these genres on the App Store.

## Conclusion

In this project, we analyzed data from the App Store and Google Play mobile apps with the goal of finding what type of apps attract more users and recommending an app profile that show potential in the market.

We concluded that an app about books could be profitable for both the Google Play and the App Store markets. 